engines/mohawk/riven_stacks/domespit.cpp

```diff
@@ -125,9 +125,25 @@ void DomeSpit::checkSliderCursorChange(uint16 startHotspot) {
 }
 
 int16 DomeSpit::getSliderSlotAtPos(uint16 startHotspot, const Common::Point &pos) const {
```

Comment:
Please rename this function to match its new role. `getSliderSlotClosestToPos` or something.